lib/wrap-request.js

```diff
@@ -1,18 +1,24 @@
 module.exports = wrapRequest
 
-async function wrapRequest (request, options, retryCount = 0) {
+async function wrapRequest (request, options) {
   const retryRequest = function (after) {
     return new Promise(resolve => setTimeout(resolve, after * 1000))
-      .then(() => wrapRequest(request, options, retryCount + 1))
+      .then(() => wrapRequest(request, options))
   }
 
   try {
     return await request(options)
   } catch (error) {
-    const maxRetries = Math.max(~~error.retries, 0)
-    const after = Math.max(~~error.retryAfter, 0)
+    const retryCount = options.request != null ? ~~options.request.retryCount : 0
+    const maxRetries = ~~error.retries
+    const after = ~~error.retryAfter
 
     if (maxRetries > retryCount) {
+      if (options.request) {
```

Comment:
`options.request` is always set, maybe check for `'retryCount' in options.request` instead

Reply:
It wasn't set in the fixture. I'll refactor this once the tests are using the real octokit/rest package.